controller/src/main/resources/org/jboss/as/controller/descriptions/common/LocalDescriptions.properties

```diff
@@ -630,8 +630,14 @@ root.composite.reply=A composite operation response that consists of all the ste
 # Common Operations
 read-config-as-xml=Reads the current configuration and returns it in XML format.
 read-config-as-xml.response=The XML form of the persistent configuration.
+read-config-as-xml.deprecated=Use the operation 'read-config-as-xml-file' instead.
```

Comment:
Harmless, so it is ok to proceed with it, it will be needed again once we deprecate the `read-config-as`